lib/routes/auth.js

```diff
@@ -45,4 +46,33 @@ router.post('/login', async (request, response) => {
   response.send(accessToken);
 });
 
+router.get('/', (request, response) => {
+  User.find()
+    .then((users) => response.json(users))
+    .catch((err) => response.json('Error:' + err));
+});
+
+router.get('/:id', async (request, response) => {
+  const userId = request.params.id;
+  try {
+    const user = await User.findById(userId);
+    response.send(user);
+  } catch (error) {
+    throw new Error(error.message);
+  }
```

Comment:
First, you are using `.then` and `.catch`, then it's `try-catch`. I wouldn't mix it.